.env.dev

```
@@ -1,9 +1,5 @@
-FLASK_APP=project/__init__.py
 FLASK_ENV=development
 DATABASE_URL=postgresql://shortener:shortener@db:5432/shortener
```

```suggestion
DATABASE_URL=postgresql://shortener:shortener@db:5432/shortener_dev
```
Since I was dumb and wrote the DB differently in the .env.dev.db file